gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
i think the idea is to make it so folks on windows can build the repo on their systems as well, even if we'll never do it ourselves